examples/model_loader/source/model_loader.cpp

```diff
@@ -101,15 +101,15 @@ class model_loader_app : public avk::invokee
 			auto idxFillCmd = newElement.mIndexBuffer->fill(newElement.mIndices.data(), 0);
 
 			// Submit all the fill commands to the queue:
-			auto fence = avk::context().record_and_submit_with_fence({
+			auto timelineSemaphore = avk::context().record_and_submit_with_timeline_semaphore({
 				std::move(posFillCmd),
 				std::move(tcoFillCmd),
 				std::move(nrmFillCmd),
 				std::move(idxFillCmd)
 				// ^ No need for any synchronization in-between, because the commands do not depend on each other.
-			}, *mQueue);
+				}, * mQueue, { vk::PipelineStageFlagBits2::eAllTransfer }, 1);
```

Comment:
```suggestion
			}, *mQueue, avk::stage::all_transfer, 1);
```

Better in terms of conciseness and formatting.

Reply:
✅